internal-export-file/export-report-pdf/src/resources/case-incident.html

```diff
@@ -0,0 +1,237 @@
+        </div>
+    </article>
+
+<article id="contents">
+    <h2>Table of contents</h2>
+    <h3>Overview</h3>
+    <ul>
+        <li><a href="#incident_details-title"></a></li>
+        <li><a href="#description-title"></a></li>
+        <li><a href="#confidence-title"></a></li>
+        <li><a href="#content-title"></a></li>
+        {% if tasks.edges %}
+        {% if tasks.edges|length > 0 %}
+        <li><a href="#task_type-title"></a></li>
+        {% endif %}
+        {% endif %}
+    </ul>
+
+    {% if entities %}
+    <h3>Entities</h3>
+    <ul>
+        {% for entity_type in entities %}
+        <li><a href="#{{ entity_type }}-title"></a></li>
+        {% endfor %}
+    </ul>
+    {% endif %}
+
+    {% if observables %}
+    <h3>Observables</h3>
+    <ul>
+        {% for observable_type in observables %}
+        <li><a href="#{{ observable_type }}-title"></a></li>
+        {% endfor %}
+    </ul>
+    {% endif %}
+    {% if case_external_refs %}
+    <h3>External References</h3>
+    <ul>
+        <li><a href="#External-References-title"></a></li>
+    </ul>
+    {% endif %}
+</article>
+
+<article id="columns">
+    <h2>Overview</h2>
+    <section>
+        <h3 id="incident_details-title"> Incident details</h3>
+        <table>
+            <tbody>
+                <tr>
+                    <th>Type</th>
+                    <th>Priority</th>
+                    <th>Severity</th>
+                </tr>
+                <tr>
+                    <td>{{ case_type }}</td>
+                    <td>{{ case_priority }}</td>
+                    <td>{{ case_severity }}</td>
+                </tr>
+            </tbody>
+        </table>
+    </section>
+    <section>
+        <h3 id="description-title">
+            Description
+        </h3>
+        <p>
+            {{ case_description }}
+        </p>
+    </section>
+    <section>
+        <h3 id="confidence-title">
+            Confidence
+        </h3>
+        <p>
+            <em>
+                This value represents the confidence in the correctness of the data contained within this case_incident.
+            </em>
+        </p>
+        <p>
+            {{ case_confidence }} / 100
+        </p>
+    </section>
+    <section>
+        {% if tasks.edges %}
+        {% if tasks.edges|length > 0 %}
+        {% for edges in tasks %}
+        <h3 id="task_type-title">Tasks list</h3>
+            <table>
+                <tbody>
+                    <tr>
+                        <th>Name</th>
+                        <th>Description</th>
+                        <th>Due Date</th>
+                        <th>Status</th>
+                    </tr>
+                    {% for edge in tasks.edges %}
+                    <tr>
+                        <td>{{ edge.node.name }}</td>
+                        <td>{{ edge.node.description }}</td>
+                        <td>{{ edge.node.due_date[:10] }}</td>
+                        <td>{{ edge.node.status.template.name }}</td>
```

Comment:
Do we always have a status ? is `status.template.name` always defined ?

Reply:
status.template.name is always defined, I think